docs/source/tutorials/qaoa.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>static_argnums=(1, 2), otherwise jax backend will be broken</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Sorry Im not quite sure what shall I do.</p><p>Could you please give a little bit more info?</p>

Reply:
see here: https://tensorcircuit.readthedocs.io/en/latest/api/backends/tensorflow_backend.html#tensorcircuit.backends.tensorflow_backend.TensorFlowBackend.jit. you need to add one argument for ``jit`` method